python-sul-2017/videos/fernando-massanori-keynote.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Keynote Fernando Massanori",
+  "recorded": "09-09-2017",
+  "speakers": ["Fernando Massanori"],
+  "thumbnail_url": "https://i.ytimg.com/vi/q4JZQXrwWo8/hqdefault.jpg",
+  "title": "Fernando Massanori - Keynote",
```

Comment:
The title of this talk is "Seja Feliz Programando em Python"
See https://youtu.be/q4JZQXrwWo8?t=17